scripts/Reflectometry/isis_reflectometry/procedures.py

```diff
@@ -5,7 +5,7 @@
 #   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
 # SPDX - License - Identifier: GPL - 3.0 +
 # pylint: disable=too-many-lines, invalid-name, too-many-arguments, too-many-branches, too-many-locals
-# ruff: noqa: F403,F405   # Wild imports and Undefined locals
+# ruff: noqa: F403,F405,E741   # Wild imports, undefined locals and ambiguous variable names
 from math import *
```

Comment:
Is this file beyond saving? It's got loads of stuff disabled. I suppose it's in the scripts folder so it's the Wild West

Reply:
Yes I believe this file would require too much effort to put to right. Also, after talking to LSS, we believe it is unused because it has a lot of references to mantidplot inside